auto_round_extension/torch/qlinear_torch.py

```diff
@@ -127,7 +127,6 @@ def pack_248_bits(self, linear, scales, zeros, g_idx=None):
 
         if isinstance(zeros, torch.Tensor):
             zeros = zeros.t().contiguous()
-            zeros -= 1
```

Comment:
help add an accuracy ut for  torch backend  in cpu ut , heng has supported evaluation with limited

Reply:
added in this PR，will update limit args.